working-groups/WORKING-GROUPS.md

```diff
@@ -66,7 +66,7 @@ The current working groups are:
 - [Security](#security)
   <!-- TODO add charters for each group -->
 
-## API Core
+## Serving API
```

Comment:
This breaks the section link above (L56), may want to update up there too

Reply:
I'd also move this down so that the list stays in alphabetical order

Reply:
oh nice catch